python/res/enkf/analysis_config.py

```diff
@@ -74,15 +88,57 @@ def __init__(self, user_config_file=None, config_content=None):
                 super(AnalysisConfig, self).__init__(c_ptr)
             else:
                 raise ValueError('Failed to construct AnalysisConfig instance from config file %s.' % user_config_file)
-        else:
+
+        if config_content is not None:
             c_ptr = self._alloc(config_content)
             if c_ptr:
                 super(AnalysisConfig, self).__init__(c_ptr)
             else:
                 raise ValueError('Failed to construct AnalysisConfig instance.')
 
+        if config_dict is not None:
```

Comment:
This will crash if there are optional parameters missing, config_dict.get() instead...

Reply:
good idea, will do so

Reply:
Decided to leave it.